.github/workflows/test.yml

```diff
@@ -81,6 +81,14 @@ jobs:
           source $VENV
           pytest tests/
 
+      - name: Run juypter tests
+        env:
+          MPLBACKEND: Agg
+        run: |
+          source $VENV
+          pytest --nbmake openbb_terminal/reports/*ipynb
+          pytest --nbmake openbb_terminal/dashboards/
+
       - name: Start Terminal and exit
```

Comment:
Let's remove this test from the CI because it makes requests that aren't mocked and it takes a lot of time